### ISSUE TITLE
Short Story Proposal - Shattered World Evacuation.

### ISSUE THREAD

**Issue by Fzzr** [endless-sky/endless-sky#7228]
### Problem Description

This is a proposal for a short-to-medium length mission string about a non-hyperspace-capable species escaping a cosmic disaster. One thing we've been looking to do for a while is add "single system" species. They add flavor to the galaxy without requiring the huge effort that a hyperdrive-capable species does.

From the [Development Roadmap](https://github.com/endless-sky/endless-sky/wiki/DevelopmentRoadmap):

- Fill the rest of the galaxy with various sized clusters of systems, with a jump drive needed to travel between them.
- More alien species.

### Related Issue Links

Other issues about this type of species we want to avoid overlapping:

- #6782 Low tech species where most of the content is about first contact.
- #7211 No-hyperdrive system that lives in a gas giant. The story is about the unintended consequences of first contact.

### Desired Solution

This mission string is about a species, yet to be named, whose world is about to be destroyed. This destruction must be predictable, foreseeable a good ways into the future, and completely unstoppable even by Pug and Drak level technology. Most stories like this in science fiction use a supernova, but I think it needs to be more than that, even. The suggestion from discord was a [cosmic string](https://en.m.wikipedia.org/wiki/Cosmic_string).

This would be a post-Jump-Drive event. The species MAY be in a hyperdrive network, but it MUST NOT be accessible without a good degree of exploration. It must also be a long distance away from the parts of the galaxy explored during and immediately after the main storyline, for reasons that will become clear in the story summary. The other interesting thing is this will be a mission string with a long but concrete in-universe timer. The result of the string is based on how much the player gets done in that amount of time, ranging from "nothing, the species is reduced to the dregs they can evacuate to their own extra-planetary settlements" to "virtually all rescued". Additionally, the things the player does will lead to changes in various places.

Story Summary:

- The player happens upon a particular system far away from inhabited space. It contains a space age but not hyperdrive-capable civilization. In the system is an Archon. It immediately contacts the player to request support with the incoming disaster.
  - If the player happens upon the system before the rest of the galaxy is ready (basically whatever stage of various campaigns we decide are necessary to make this work) they are given a standard Archon warning off. In this case, the string kicks off once the world is ready, either by a new visit or a summons from the Archon somehow.
- The impending disaster is the total annihilation of the highly populated homeworld of the species. The population is such that the species has no viable way to evacuate to other habitable locations, even if they had enough space to move them to. This is a population on the scale of billions.
- The support needed is: Help find a new home for the species, and a way to move as many of them to the new place, as fast as possible.
- This is something the Drak/Archons cannot do themselves - it requires the support of a civilization to physically move that many people.
- The player now faces two challenges: Where to put them, and how to get them there.
- To get them there, only the Quarg have an arbitary number of jump-capable ships, but even they don't have the numbers to evacuate billions in the short time period available.
- Therefore, there is only one true option: Get the support of the Pug to open a wormhole between the doomed system and wherever they will evacuate to. This requires a truce among the Pug, Quarg, and Archons (maybe tricky in lore?).
- For where to put them, the options are:
  - Quarg ringworld. Logistical support depends which one we justify them going to in story. If it was possible to do the one in human space, that would allow the humans to be the logistical support, but that raises serious questions in the worldbuilding space. In any case the Quarg themselves can do better logistically if they're using a wormhole to transport refugees instead of having to jump all the way across the galaxy.
  - Coalition (extra space in a ringworld? uninhabited planet?) - Logistical support from the Coalition or some fraction of it. More than 1/4 of the species can be saved this way. Concerns here are about the conflicts in a Coalition storyline.
  - Uninhabited planet in the Ember Waste - Logistical support from the Remnant. Only at most 5% of the species can be saved this way. @Zitchas Please comment if this is possible within their culture.
  - Direct immigration to an already inhabited world in Hai space - logistical support from the Hai and humans who already know about the Hai. Up to 1/4 of the species can be saved this way. This is probably the easiest to justify in lore.
  - While the Wanderers would not likely be able to take in refugees due to their own evacuation, they would be interested in saving the biodiversity of the homeworld. This is a stretch, I'm just looking for a way to get as many involved as possible. Viability depends on where the story stands, as Wanderer resources are stretched thin.
- The player is not limited to one option - the more there are, the more of the species can be saved. They just need to know that those who go to each space will not be able to maintain physical contact with the other locations for the long-distant future.

The ending needs to scale in two ways - scope of change to the universe and degree of reward to the player. Given the limited time available, it should be possible but not easy to get all possible evacuation sites set up early enough. If none are set up there could be some punishment, but from a gameplay perspective deciding to ignore a mission shouldn't be *too* harmful, so it can be more emotional or small-scale practical (like the Archon of that system being permanently hostile to the player). Each location that receives refugees will experience some change due to the presence of the refugees. If enough of them are saved (say 80%) there can be some larger, unique reward. Regardless of how many are saved, the survivors will view the player as the most important single being in the history of their species.

### Alternative Approaches

If the full story turns out to be too difficult to fit within lore, we can still have a system with such a species in the aftermath of the disaster (ie living in only a few colonies they were able to settle in the limited time they had, leaving behind billions to die.)

### Additional Context

Required moving parts:

- [ ] Worldbuilding for the species
- [ ] Map location such that it does not interfere with other ongoing work
- [ ] Working out of lore questions and impact on other storylines/species
- [ ] Actual plot outline
- [ ] Writing and event scripting
- [ ] World end state changes (news etc.)

Art assets:

- [ ] Pre-shattered world
- [ ] Post-shattered world
- [ ] A few ships for the species themselves
- [ ] New images for any worlds that go from unsettled to settled
- [ ] Possibly something to do with the best-ending reward

**Comment by Zitchas** [comment]
Thoughts:
1. This is an extremely ambitious project, the sort of thing that any of our "mortal" civilizations (aka everyone except Pug, Quarg, and Drak) could handle only on a total-war style effort. As in the entire civilization devotes its energy to accomplishing the task. That's not a small mission chain.
2. We do have the Pug, who at one point moved a ring world, so might be able to move a planet, and could probably open a wormhole. (Just to clarify: They have the tech to do wormholes, but we have no idea how much effort that takes. Could be a snap of the fingers for them, or maybe each wormhole has to be setup and takes a couple years of work before they can activate it. The moving ringworlds I suspect takes civilization levels of effort to achieve, and probably at significant cost)
3. We also have the Quarg, who are the only species who could reasonably field enough ships to move billions of people (plus support infrastructure) in any reasonable frame of time.

From my perspective, there's a choice to be made: Either:
A: have the campaign focus on getting the Pug and Quarg to be on "not-shoot-on-sight" terms for a short duration (or at least in a certain area) long enough to do the evacuation; or
B: have it be on a massively longer time scale, such that it does not require "total war" levels of industrial dedication from the rest of the galaxy to handle. This *probably* means many years, possibly decades.


As for the Remnant's involvement... They lack the fleet to move any significant fraction of the population, or the planet(s) to put them on. At best they'd be evacuating a few tens of thousands of people so the species could go on. They'd probably be willing to do so, too, but not to their own worlds. They'd probably prefer to just relocate them to the nearest habitable world. A more likely involvement for the Remnant would be informational: They're probably the most likely faction to *hear* any distress call, and they'd definitely be the ones most interested in watching the stellar calamity unfold. So they'd probably be better suited to be the group pushing the player to talk to different groups or whatnot. They are very well aware that the player is known to many alien factions, and are quite comfortable letting the player take the glory of saving the alien race. They'll just stick to the shadows and help the player know what they need to know to help everyone else.

**Comment by Arachi-Lover** [comment]
I'm sorry, but A cannot happen. It's tremendously naive to think the Pug and Quarg would be on any form of terms where they can work together after the history between them. If such a truce were to happen, it'd be far after the game's timespan.

As for the Coalition, while they do have the ships to transport a good chunk of such a population in a timely manner, the resources to support the added population to their area, and the worlds to house the population (37 uncolonized planets/moons in Coalition space currently in the goldilocks zone), the Heliarchs wouldn't risk the JDs they have treking across the galaxy to give this species a ride. Leaving the Coalition at all is a tremendous taboo to them.
I also don't want the Pug to be involved in the Coalition, so them opening a wormhole directly to the Coalition isn't desireable either.
Basically the problem with them is that there's no easy solution as to how the Heliarchs get to this species in the first place.

**Comment by Fzzr** [comment]
OK, after a LOT of discussion on discord, here's an updated plan.

- Archon/Drak are gone from the plan. The player finds out from the Remnant, the Pug, or the species themselves.
- The Pug are involved in providing wormholes, but nothing else. Reason: Pug reasons. They want to make the younger races advance through adversity again.
- The Quarg agree to stand down and allow temporary wormholes from the Pug, but will not interact directly due to lack of trust.

Following this, here are the potential ways for each civilization to interact, in increasing order of both importance and difficulty.

1. Humans. In the minimal case, the project is just about humanity coming together to save a doomed species. This can therefore happen any time after the war, and the species can be anywhere JD accessible. Regardless of future options, the humans can dedicate a lot of resources and a lot of space.
2. Humans + Hai. After the Hai and Humans are fully in contact, it becomes possible for the project to be cooperative between them. In this case, the Hai can take as many as they can, but their space is mostly full so the refugees would be spread very widely across their worlds.
3. Wanderers. They cannot take refugees due to their own crisis, but they would be willing to rescue biodiversity. This is complicated because it advances the degree of diplomatic contact between human and Wanderers - it's a temporary non-JD link between their respective hyperlane networks. 
4. Remnant. The Remnant have hardly any logistical capacity, but they would have deep empathy for a species fleeing extinction. They would be able to help a small number (tens to at most a few hundred thousand) settle a single, nearly barren world - enough to create a viable colony, but no more. The challenge for them is letting them help while preserving their secrecy from everyone else.
5. Coalition. They have massive logistical capacity, but they have no trust of the elder races. They would not accept a Pug wormhole into their space. In their case, they would agree to ferry as many refugees as possible from human space, probably Poisonwood, to their own space by JD. In this case there ends up being a second human space wormhole to Poisonwood, and as a consequence there are two settlements in human space in the end - Poisonwood and larger one elsewhere. The challenge here is this exposes the Coalition to the public of humanity for the first time, which is a lot of impact for a medium mission string to have. Without a lot of heavy lifting it exposes them to everyone else, too.

Since I didn't include them in the original, from a thematic perspective here's what the consequences are to each potential place they go.

- **Humans**: Humans can take by far the largest group of refugees. They will be able to settle in mostly the a single location, so preserving their culture will be very possible. However, they will suddenly become a minority in a very large space, and not everyone will welcome them unconditionally.
- **Hai**: The Hai do not have lots of space on any one world for them, so moving to Hai space means leaving a joint culture behind almost completely. Hai space is safe, and they will be welcome, but connection to the wider culture is almost totally lost.
- **Wanderers**: Not relevant to refugees choice. 
- **Remnant**: To go with the Remnant is to give up every comfort civilization offers and start from nothing. However, they will be independent and free, in no way subject to another culture, and with ample space to grow.
- **Coalition**: They will treat the refugees as a subject species, essentially confining them to a single world and not granting anything other than very local autonomy - but they *wiil* be safe.
- **Stay home**: There are a few space colonies that will survive the disaster. Some but not many can stay behind. Their species will carry on, and they will be truly the master of their own destinies - even as they survive the greatest tragedy that can ever happen.

**Comment by roadrunner56** [comment]
If I can offer my own two cents:
I really like this idea, but as others have said it is a big story and lore commitment. I think that it's plausible that all species listed can be involved in the effort, but the Coalition would be the hardest. If you don't want the Coalition exposed, then I like the idea of some refugees being brought to Poisonwood, and then the player taking a large fraction of those refugees and bringing them to the Coalition. This ensures that the Coalition does not have to risk their jump drives or expose them to the rest of humanity, while still getting them involved. 
Otherwise, I agree with everything you said in your comment.

**Comment by Fzzr** [comment]
OK, a *lot* more discussion on Discord led to no consensus on a lore-friendly way to involve any of the elder species. Since changes in hyperspace links and wormholes thus become out of the question, a new way to provide a civilization-scale challenge was needed. The new proposal allows the scope to be only as big as needed, possibly human only, thus avoiding impact or dependency on any other storyline (except potentially Hai Revealed)

1. Any time after the war ends, the player can receive a landing or spaceport mission in the Deep.
2. A scientist has made their life's work to study a particular, quirky star. It's a very strange system, moving at an unbelievably fast rate through the galaxy, perpendicular to the galactic plane - in other words, it's almost certainly extragalactic in origin. On top of this, it's about to be in range of <space hazard, fill in later> which will leave the system devastated. They want you to take them there by Jump Drive as soon as possible, so they can see it before that happens.
3. It's just one jump away from the Deep (if possible, make it only visible once the mission is given) so the player agrees and takes them there.
4. When you arrive, you find that on top of being extragalactic and about to be destroyed, it's also inhabited.
5. There's no way to save more than a pitiful handful of people by ferrying them over one jump at a time. (Note: easter egg, if the player has 100,000+ bunks somehow, they actually can move a lot of them on their own.) Luckily, there's no known upper limit on the size of ship a jump drive can move. Therefore, humanity (and maybe others) will build worldships, capable of moving people by the hundreds of thousands or more, to get the most throughput possible from the few Jump Drives the player can provide.
6. So now the player is faced with a challenge: they have an un-contacted species mere [amount of time TBD] away from extinction, and saving them is a problem that requires cooperation from basically all of human civilization.
7. The other challenge is finding a place for the refugees to live. This can be a wide open problem. Certainly the Deep will be the first landing place, but they can't hold them all, so they need options throughout human space and possibly beyond. Different places will be willing to accept different numbers of refugees, under varying conditions and for different reasons.
 - The Deep will have some space available with little persuasion, but it's not enough alone.
 - The Syndicate can build worldships and find space for a price. Probably a bunch of negotiation, or you can short-circuit the whole thing by offering them a billion credits.
 - The Free Worlds need to find a place that can both support them and where they won't cause too much instability, but it can be done.
 - The Republic will do a bunch of politics and ultimately offer a spot in the Dirt Belt - possibly without asking everyone in that place for consent first. Alternatively, they show they learned their lesson from the war and do some actual democracy.
 - Assuming this is after Hai Revealed, this is a chance for them to get some real external stimulus, and they definitely have the industrial capacity to put together at least a few worldships of their own.
 - Some refugees may request a new world of their own, to start from scratch. This will require the player to spend time scouting, but if the game can handle it, maybe you can name it.
 - [Storyline allowing] The Wanderers can't take refugees, but if you can provide them a worldship, they will save as much as they can of everything else.
 - [Storyline allowing] The Coalition will take some refugees - but they do it entirely on their own terms, you provide the jump drives, and they keep the jump drives when it's all over.

In the end, lots of different places will have some amount of change. At the end all the worldships will be scrapped, as they're completely impractical. I'd like to let the player keep one, but I don't think it's possible for something to feel like a real worldship while also being balanced.

**Comment by Fzzr** [comment]
As for the actual dispositions of the elder species:

- These unlucky folks do not have a Pug group following them around doing low-touch intervention, and any you talk to should basically look at you blankly.
- This type of active intervention is not how the Drak/Archons work
- The Quarg will look at them as an extragalactic, and thus not worth intervention (just like the Wanderers were at first). However they can be persuaded to offer technical support to worldship design, as long as it doesn't involve technology transfer. Note that too much Quarg involvement would cause the Coalition to refuse to participate at all. If we want a way to exclude the Coalition without making them seem heartless, this can be the reason.

**Comment by dwRchyngqxs** [comment]
I remember Korath (Kor Efreti) having "world ships" able to transport thousand of crew, maybe this could help evacuating a planet?

**Comment by roadrunner56** [comment]
The Korath Exiles are generally strapped for resources, and don't really have any ships to spare, especially not their largest civilian ones. In addition, they're not really on speaking terms with the player. For that reason, they're effectively out of this plotline.

**Comment by dwRchyngqxs** [comment]
> The Korath Exiles

I did not mean Korath Exiles (I even specified Kor Efreti), those are just not the kind of species to save another one but rather to plunder them to extinction. I'm talking about the Korath you encounter during the wanderer storyline, which call themselves "Korath friends" and are willing to trade with the player.

**Comment by roadrunner56** [comment]
Ahh, I shouldn't comment when I'm tired. I meant the Kor Efreti in my above comment, apologies. As of now there aren't any ways to speak with the Efreti beyond pointing and gesturing, which probably won't be able to convince them to give up one of their valuable world ships to a species they have 0 relations with.

**Comment by ravenshining** [comment]
A massive, dramatic, direct intervention, only on the very cusp of annihilation?  Mobilising some sort of conveyance to carry millions to a new home?  That sounds to me exactly like how the the Archons at least *once* intervened, and they might enjoy the chance to elbow out the Pug and do this their style.  They might just need some cheering up; if they were to see some success with one of their existing projects -- like, oh, the Korath, say -- maybe they'd feel inspired to act again.

It's true that the Korath -- Exiles or Efreti -- are too strapped for resources to offer more help than the Remnant could on their own -- certainly not while the Remnant is fighting them.  However, I have been building storylines to make the Exiles into a faction you can get on good terms with.  Should a player have taken this path, they may have seriously angered a number of other factions, limiting their choices.

Certainly the Exiles would empathise with the idea of these refugees forever losing their home.  And while they may not have resources, they do have knowledge of how to survive *without* so much as a planet under your feet.  Due to their alien psychology, any help they offer would be *highly* conditional -- while not demanding subservience, effectively they'd need this society to swear loyalty to them -- but provided such conditions are met, they'd do all they could.  Failing those conditions, *then* they'd just use the opportunity to plunder as alleged.

In to-be-developed post- Exile-Remnant war situation, where the player has lost many of the friends they would have had in a normal playthrough, perhaps we might add a branch involving Korath.  Obviously that's not something that's practical now, but stick a pin in it, it might be a fun route to take in the future.

**Comment by Amazinite** [comment]
Copying from my message on #6782:

> It's... the case that these days, new story suggestions are typically better fit for discussion threads here on Github, or discussed on Discord, rather than having open issues sit around with story ideas that may or may not be executed upon.
